lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-toast-container {
+  position: sticky;
+  left: calc(100% - 500px);
+  height: max-content;
+  width: 500px;
+  z-index: 1;
+  overflow:hidden;
+}
```

Comment:
Right now when a button is behind the toasts but partially visible through the gap between toasts, I cannot click on it; the toasts and/or toast container receive the click instead.  This can be solved with a combination of the `pointer-events` CSS property and adjustments to the margins on toasts (i.e. make it so that the `.nx-toast` has the margin, not the `.nx-alert`).

Reply:
> Right now when a button is behind the toasts but partially visible through the gap between toasts, I cannot click on it; the toasts and/or toast container receive the click instead. This can be solved with a combination of the `pointer-events` CSS property and adjustments to the margins on toasts (i.e. make it so that the `.nx-toast` has the margin, not the `.nx-alert`).

Updated here: [864016e](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/864016ecae4dffc327694a354eae8f5dfa5eee01)